components/Chat/ChatInput.tsx

```diff
@@ -358,7 +428,30 @@ export const ChatInput = ({
               onClose={() => setIsModalVisible(false)}
             />
           )}
+
         </div>
+
+        <div className="charLimitDisp">
+          {isHighCharacterCount && (
+
+            <span className="text-orange-500">
+              approx. characters left in conversation context:
+              {(selectedConversation.model.tokenLimit * CHARACTERS_PER_TOKEN) - ((selectedConversation?.tokenLength * CHARACTERS_PER_TOKEN) + content?.length)}
+            </span>
+          )}
+
+          {isPastCharacterCount && (
+            <span className="text-red-500">
+              this conversation is past the context limit. approx. characters over:
+              { ((selectedConversation?.tokenLength * CHARACTERS_PER_TOKEN) + content?.length) - (selectedConversation.model.tokenLimit * CHARACTERS_PER_TOKEN)}
+            </span>
+          )}
+
+          { (isPastCharacterCount || isHighCharacterCount) && (
+            <span className="helpCircle" title="Once past the context limit, the conversation will no longer produce responses relevant to content before the limit">&nbsp;&nbsp;?&nbsp;&nbsp;</span>
+          )}
+        </div>
```

Comment:
<img width="1305" alt="Screenshot 2025-03-17 at 3 06 25 PM" src="https://github.com/user-attachments/assets/a9aef9dd-8d4c-4a41-a20a-8ab0a7fc2d24" />

FYI here's how it appears.